que.go

```diff
@@ -216,46 +216,56 @@ func (c *Client) LockJob(queue string) (*Job, error) {
 	}
 
 	j := Job{pool: c.pool, conn: conn}
-	err = conn.QueryRow("que_lock_job", queue).Scan(
-		&j.Queue,
-		&j.Priority,
-		&j.RunAt,
-		&j.ID,
-		&j.Type,
-		&j.Args,
-		&j.ErrorCount,
-	)
-	if err != nil {
-		c.pool.Release(conn)
-		if err == pgx.ErrNoRows {
-			return nil, nil
+
+	for {
```

Comment:
Ok, if it's that common under extremely high concurrency, 10 seems like a perfectly reasonable choice :+1: